package.json

```diff
@@ -7,18 +7,18 @@
     "documentation": "./bin/documentation.js"
   },
   "dependencies": {
+    "@babel/core": "^7.0.0-beta.40",
+    "@babel/generator": "^7.0.0-beta.40",
+    "@babel/preset-env": "^7.0.0-beta.40",
+    "@babel/preset-react": "^7.0.0-beta.40",
+    "@babel/preset-stage-0": "^7.0.0-beta.40",
+    "@babel/traverse": "^7.0.0-beta.40",
+    "@babel/types": "^7.0.0-beta.40",
     "ansi-html": "^0.0.7",
-    "babel-core": "^6.26.0",
-    "babel-generator": "^6.26.0",
     "babel-plugin-system-import-transformer": "3.1.0",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1",
-    "babel-traverse": "^6.26.0",
-    "babel-types": "^6.26.0",
-    "babelify": "^8.0.0",
-    "babylon": "^6.18.0",
+    "babelify": "git+https://github.com/ylemkimon/babelify.git#master",
```

Comment:
The official babelify project supports Babel v7 now: https://github.com/babel/babelify/blob/v9.0.0/package.json#L16